src/swift/flwr/Sources/Flower/Common/Parameter.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  Parameter.swift
+//  FlowerSDK
+//
+//  Created by Daniel Nugraha on 09.01.22.
+//
+
+import Foundation
+import NumPySupport
+import PythonSupport
+import PythonKit
+import CoreML
+import NIOCore
+import NIOPosix
+
+let appDirectory = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first!
+
+@available(iOS 14.0, *)
+public class ParameterConverter {
+    private var np: PythonObject?
+    
+    /// The permanent location of the numpyArray.
+    private var numpyArrayUrl = appDirectory.appendingPathComponent("numpyArray.npy")
+    private var group: EventLoopGroup?
+    
+    public static let shared = ParameterConverter()
+    
+    private init() {
+        initGroup()
+    }
+    
+    private func initGroup() {
+        if group == nil {
+            print("Opening Python event loop group")
+            group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+            
+            let future = group?.next().submit {
+                PythonSupport.initialize()
+                NumPySupport.sitePackagesURL.insertPythonPath()
+                self.np = Python.import("numpy")
+            }
+            
+            do {
+                try future?.wait()
+            } catch let error {
+                print(error)
+            }
+        }
+    }
+    
+    private func multiArrayToNumpy(multiArray: MLMultiArray) -> PythonObject? {
+        let pointer = try! UnsafeBufferPointer<Float>(multiArray)
```

Comment:
Won't this crash the app if it does not work out? Would it be reasonable to rewrite it, so the user gets a proper error log?

Reply:
It will not crash the app, instead it sends a nil (null) value which will be checked by the caller. Will add a log message.

Reply:
Understood! Thank you.